tests/foreman/ui/test_role.py

```diff
@@ -660,113 +654,156 @@ def test_positive_disable_all_filters_overriding_option(self):
+            2. Create more than one overridden filters in role with org B and
+                Location B
+            3. Save above role with filters
+            4. Click on 'Disable all filters overriding' button in filters
                 table in role
-            4. Login with user and attempt to access resources
 
-        :expectedresults:
-
-            1. On disable, the overridden mark is disabled for all the
-                overridden filters in role
-            2. On disable, User should have access to resources in taxonomies
-                (orgnizations and locations) assigned to role
-            3. On disable, User shouldn't have access to resources in
-                taxonomies mentioned in filter
-
-        :caseautomation: notautomated
+        :expectedresults: On disable, the overridden mark is disabled for all
+            the overridden filters in role
         """
 
     @stubbed
-    @tier2
-    def test_positive_create_org_admin(self):
-        """Create Org admin role which has access to all the resources within
-        organization
+    @tier1
+    def test_positive_create_org_admin_from_clone(self):
+        """Create Org admin role which has access to most of the resources
+            within organization
 
         :id: 03f41736-c5c5-414a-ab75-650cecd6f6cd
 
         :steps:
 
-            1. Clone Manager role which has most resource permissions
-            2. Assign taxonomies (organizations and locations) to the cloned
-                role
-            3. Add more missing resource permission to the cloned role to make
-                it Org Admin having access to all resources
-            4. Create user and assign org admin role
-            5. Login with user and attempt to access resources
-
-        :expectedresults:
+            1. create Org Admin by cloning 'Organization admin' role which has
+                most resources permissions
 
-            1. Successfully created Org Admin by cloning manager role
-            2. Successfully assigned taxonomiess to role
-            3. Missing resource filters are added successfully to the Org Admin
-                role
-            4. User is able to access all the resources, permissions only in
-                taxonomies selected in org admin role
-            5. User shouldnt be able to access resources, permissions in
-                taxonomies not selected in org admin role
+        :expectedresults: Org Admin should be created successfully
 
         :caseautomation: notautomated
         """
 
     @stubbed
     @tier1
     def test_positive_create_cloned_role_with_taxonomies(self):
-        """Test taxonomies can be set on cloned role.
+        """Taxonomies can be assigned to cloned role
 
         :id: ad20f5c7-3df7-4b43-8a52-097c87676d07
 
         :steps:
 
-            1. Attempt to clone any existing role(e.g Manager Role)
+            1. Create Org Admin by cloning 'Organization admin' role
```

Comment:
The org admin isnt a role, but we will create a role that will have all the permissions for perticular org(s) and location(s).
In short, Org Admin is shortname for Admin that controls the Org.